### DISCUSSION TITLE
Right clic in song select should fast browse like stable

### DISCUSSION THREAD

**Discussion by Biokax** [ppy/osu#18286] [Ideas] [answered]
With a large amount of beatmaps the right clic to browse through songs on stable is a must.
The same mecanic should also apply in every interface like beatmaps browse

https://user-images.githubusercontent.com/50368878/168485377-da223b7f-dd69-4b37-9adc-5b22287fdbaf.mp4

**Comment by bdach** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/20418176/168485645-8e3f917b-7df7-4a70-aed1-762e398038fb.png)

**Reply by Biokax** [reply]
I missed this button thanks, this logic could also apply in song browse ?
